Manual/manJHUGenerator.tex

```diff
@@ -1375,14 +1378,54 @@ \section{JHU Generator Coupling Converter (JHUGenLexicon)}
 Lambda_z1=100 Lambda_w1=100 Lambda_zgs1=100
 \end{verbatim}
 
-\section{Modified Zff couplings in MCFM}
-We support modified Zff couplings in certain processes in the standalone JHUGen-MCFM package of the JHU Generator Framework. We allow modified Zff couplings both production and decay for $gg \to 4l$ processes. Zff couplings are set separately for production and decay.
-In production, we parameterize the modifications to the Zff couplings in production as shifts to the SM coupling of the Z-boson to up-type and down-type quarks. In decay, we allow the user to set the exact value of the Zff couplings to leptons and quarks. To utilize these features one must install the required package
+\section{Modified Zff couplings}
+\subsection{JHUGen}
+We support modified Zff couplings in all on-shell Spin-0,Spin-1 and Spin-2 decays in JHUGen. Modifications to Zff couplings for associated Z in process 50 and 51 is not currently supported.
```

Comment:
Spin-0, Spin1

Reply:
This process does work for Spin-2 decays, is it because you want Spin-1 to be written as Spin1? I see in other places in the paper it is written as spin-1.

Reply:
Ah I see you wanted a space after the comma